app/modules/node.c

```diff
@@ -21,6 +21,39 @@
 #define CPU80MHZ 80
 #define CPU160MHZ 160
 
+#define DELAY2SEC 2000
+
+static void restart_callback(void *arg) {
+  UNUSED(arg);
+  system_restart();
+}
+static int default_onerror(lua_State *L) {
+  static os_timer_t restart_timer = {0};
+  /* Use Lua print to print the ToS */
+  lua_settop(L, 1);
+  lua_getglobal(L, "print");
+  lua_insert(L, 1);
+  lua_pcall(L, 1, 0, 0);
+  /* One first time through set automatic restart after 2s delay */
+  if (!restart_timer.timer_func) {
+    os_timer_setfn(&restart_timer, restart_callback, NULL);
+		os_timer_arm(&restart_timer, DELAY2SEC, 0);
```

Comment:
Whitespace

Reply:
Noted. Will fix

Reply:
@nwf,
I've just done a `git diff 06aa248f bd9b4253 | gedit -` (the patch for this PR) and did a global highlight for `\t`.  This whole tab issue is a PITA.  The Lua style uses an indent depth of 2 chars and always uses two spaces to per indent.  It does use tabs (with a default of 8 chars per stop)

However modules are all over the place.  Some use tab indents (and some assume a tab stop of 4) so it is extremely easy to get the visual alignment screwed.  My vote is that we should address this as a module-wide cleanup rather piecemeal one-offs.

Reply:
>  Some use tab indents (and some assume a tab stop of 4) so it is extremely easy to get the visual alignment screwed. 

#1168, created over 4y ago.

Reply:
Yup.  Golden oldies that still bite.